src/Response.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Utopia\Fetch;
+
+/**
+ * Response class
+ * This class is used to represent the response from the server
+ * @package Utopia\Fetch
+ */
+class Response
+{
+    /**
+     * Response Body
+     *
+     * @var string
+     */
+    private string $body;
+    /**
+     * Response Headers
+     *
+     * @var array<string, string>
+     */
+    private array $headers;
+    /**
+     * Response Status Code
+     *
+     * @var int
+     */
+    private int $statusCode;
+    /**
+     * Response Method
+     *
+     * @var string
+     */
+    private string $method;
```

Comment:
This method refers to the request method for which we are building the response. I just though it might be useful in some usecases, not sure though, should I remove it?

Reply:
okay, got it!